src/components/themes.cy.tsx

```diff
@@ -0,0 +1,109 @@
+"use client";
+
+import React from "react";
+import styled, {
+    DefaultTheme,
+    rainbowPalette,
+    standardPaletteList,
+    standardPalette,
+} from "styled-components";
+import StyleManager, { lightTheme, darkTheme } from "@/app/themes";
+import { Result } from "axe-core";
+
+const checkColorContrast = (): void => {
+    const terminalLog = (violations: Result[]): void => {
+        cy.task(
+            "table",
+            violations.map(({ id, impact, description, nodes }) => ({
+                id,
+                impact,
+                description,
+                length: nodes.length,
+            }))
+        );
+    };
+
+    cy.injectAxe();
+    cy.checkA11y(undefined, { runOnly: { type: "tag", values: ["wcag2aa"] } }, terminalLog);
```

Comment:
can we generalise the existing `checkAccessibility` or write a new `checkColourContrast` function in Cypress, rather than copying and pasteing the commands we want to run?

Reply:
I think I prefer not to do this because the test should be 'easily readable' - as much of the implementation details of 'exactly how it checks accessibility' should be a Cypress thing... but if others disagree, I'm fine for it to stay!

Reply:
Makes sense - seems like it should be treated as something "controlled at the back by cypress" and shouldn't be configured in a test file - so I have moved it to cypress to use `cy.checkColorContrast()` instead